plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -120,6 +120,19 @@ export class PersonState {
         this.updateIsIdentified = false
     }
 
+    async mergeBlocked(reason: string, details: Record<string, any>): Promise<void> {
+        // This class abuses it's access to event and adds $merge_blocked_reason to properties if the merge is blocked
+        this.event.properties = { ...(this.event.properties || {}), $merge_blocked_reason: reason }
+        status.warn('🤔', `refused to merge ${reason}`)
```

Comment:
Is logging this really desired? If so, I think it should include a lot more info like the team and distinct_ids

Reply:
just moved that code around, we can remove the logging